lib/versionbot.ts

```diff
@@ -85,20 +85,20 @@ export class VersionBot extends GithubBot.GithubBot {
             {
                 events: [ 'pull_request' ],
                 name: 'CheckVersionistCommitStatus',
-                suppressionLabels: [ 'flow/no-version-checks' ],
+                suppressionLabels: [ 'procbots/versionbot/no-checks' ],
                 workerMethod: this.checkVersioning
             },
             {
                 events: [ 'pull_request' ],
                 name: 'CheckVersionistCommitStatus',
-                suppressionLabels: [ 'flow/no-version-checks' ],
-                triggerLabels: [ 'version/check-commits' ],
+                suppressionLabels: [ 'procbots/versionbot/no-checks' ],
+                triggerLabels: [ 'procbots/versionbot/check-commits' ],
                 workerMethod: this.checkVersioning
```

Comment:
Independent of this change, but I don't understand why we have two `CheckVersionistCommitStatus` blocks here, which look identical except the second requires triggering. What does the second do that the first doesn't already cover?